### PR TITLE
Allow additional scopes to be added to Google Authentication module

### PR DESCRIPTION
**Pull request by jamiesnape**
@cpatrick @mgrauer PTAL.

### CODE REVIEW

**Comment by jamiesnape** [comment]
@cpatrick @mgrauer PTAL.

File: modules/googleauth/Notification.php [hunk 1]
```diff
@@ -38,69 +48,128 @@ public function init()
...
+        $interval = new DateInterval('P1M');
+        setcookie(
+            GOOGLE_AUTH_ACCESS_TOKEN_COOKIE_NAME,
+            null,
+            $date->sub($interval)->getTimestamp(),
+            '/',
+            $request->getHttpHost(),
+            (int) Zend_Registry::get('configGlobal')->get('cookie_secure', 1) === 1,
+            true
+        );
```
Comment (cpatrick) [inline modules/googleauth/Notification.php]: This seems like the same code as above modulo an arg or two?

File: modules/googleauth/controllers/CallbackController.php [hunk 2]
```diff
@@ -145,35 +109,46 @@ protected function _createOrGetUser($info)
...
+        );
+        setcookie(
+            GOOGLE_AUTH_ACCESS_TOKEN_COOKIE_NAME,
+            $client->getAccessToken(),
+            $expires,
             '/',
             $request->getHttpHost(),
             (int) Zend_Registry::get('configGlobal')->get('cookie_secure', 1) === 1,
             true
         );
```
Comment (cpatrick) [inline modules/googleauth/controllers/CallbackController.php]: Again with this block of code :smile_cat:

File: modules/googleauth/Notification.php (hunk 1 above)
Comment (jamiesnape) [inline modules/googleauth/Notification.php]: That is true. I will refactor.

**Comment by cpatrick** [comment]
Once the cookie setting is re-factored into a helper function to reduce duplication, LGTM.

**Pull request merged**